src/redux/remoteData/asyncActionCreator.ts

```diff
@@ -145,30 +201,28 @@ export const fetchBrowserGeolocation = () => {
   };
 };
 
-export const startup = () => {
-  return (dispatch: Dispatch<ApplicationState>,
-          getState: () => ApplicationState) => {
-    // dispatch donations
-    dispatch(fetchDonations());
-    dispatch(setUiState(LocationUiState.FETCHING_LOCATION));
-    const state = getState();
-    // clear contact indexes loaded from local storage
-    dispatch(clearContactIndexes());
-    // add completed issues from Choo app in localStorage to
-    // this apps callState.completedIssueIds
-    migrateLegacyCompletedIssues(dispatch);
-    const loc = state.locationState.address;
-    if (loc) {
-      // console.log('Using cached address');
-      dispatch(fetchAllIssues(loc))
-        .then(() => {
-          setLocationFetchType(LocationFetchType.CACHED_ADDRESS);
-        });
-    } else {
-      dispatch(fetchBrowserGeolocation());
-    }
-    dispatch(fetchCallCount());
-  };
+export const startup = (dispatch: Dispatch<ApplicationState>,
```

Comment:
I'm not sure why this declaration changed from the `redux thunk` structure of a function that returns a function to a standard function call.

Reply:
Probably because I messed around with the right way to do a bunch of times. Will most likely return to the way it was before and then just dispatch `startup` before `rehydrated` as you suggested!